### ISSUE TITLE
Batch norm statistics for COCO dataset

### ISSUE THREAD

**Issue by ayooshkathuria** [pjreddie/darknet#647]
I'm trying to port the code to PyTorch for sake for creating an object detector (No training). The problem is the population statistics (mean, and variance) of the built-in PyTorch batch norm layer don't play nice with the v3 weights and lead to a NaN's in the output. Can you somehow share the population statistics for YOLO v3 trained on COCO? @pjreddie
